data/bson-corpus/binary.json

```diff
@@ -39,6 +39,12 @@
             "canonical_bson": "1D000000057800100000000473FFD26444B34C6990E8E7D1DFC035D400",
             "canonical_extjson": "{\"x\" : { \"$binary\" : {\"base64\" : \"c//SZESzTGmQ6OfR38A11A==\", \"subType\" : \"04\"}}}"
         },
+        {
```

Comment:
`bson_corpus_spec_test` actually just runs the json files in `data` (edited above), so changes to this file might be unnecessary; just thought I'd update both in case.

Reply:
Awesome, thanks so much for making that ticket!

Reply:
1840 should be completed with a recent commit on this PR. Removed all `data/*.json` files and changed the `bson_corpus_spec_test` to use `data/bson-corpus`. I did not update the spec tests using `./etc/update-spec-tests.sh`, but let me know if I should.

Reply:
That sounds good!